theme/static/style/style.css

```diff
@@ -325,52 +323,97 @@ pre code {
 
 /* Footer Section */
 
-.footer{
+.footer {
     padding: 2% 0;
     display: inline-block;
     font-size: 1.2rem;
 }
 
-.footer-logo{
+.footer-logo {
     float: right;
     margin-right: 1rem;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright{
+.footer-copyright {
     line-height: 1.8em;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright a{
+.footer-copyright a {
     text-decoration: underline;
 }
 
-.footer-sociallinks{
+.footer-sociallinks {
     list-style-type: none;
     text-align: right;
 }
 
-.footer-sociallinks li{
+.footer-sociallinks li {
     display: inline-block;
     margin-right: 1.5rem;
 }
 
-.footer-sociallinks a{
+.footer-sociallinks a {
     font-size: 3.5rem;
 }
 
+/* Features Page */
+
+.horizontally-stacked-list {
+    display: flex;
+    float: left;
+}
+
+.left-floating {
+    float: left;
+}
+
+.testplan {
+    float: right;
+    clear: right;
+    margin-bottom: 5rem;
+}
+
+.testplan-img {
+    margin-bottom: 5rem;
+}
+
+.mark-and-status-bugs {
+    float: left;
+    margin-bottom: 5rem;
+}
+
+.call-to-action-field {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    text-align: center;
+}
+
+.about-panel {
+    padding-top: 2rem;
+}
+
 @media screen and (max-width: 1360px){
     /* Screenshots section */
     .screenshots .col-2 {
         display: none;
         width: 0;
     }
 
-    .screenshots .col-8 {
-        width: 100%;
+    /* Features Page */
+    .features .col-2 {
+        display: none;
+        width: 0;
+    }
+    .features .col-8 {
+        width: 97%;
+    }
+    .features .margin0 {
+        margin-left: 7.5%;
+        width: 87%;
```

Comment:
Use wid/max-width 100% and margin in rems, 1.8 is the default everywhere else I believe. You can also use box-sizing: border-box to make everything fit inside the container.

Reply:
box-sizing: border-box doesn't work for me, the images go out of the screen due to the margin, that is why I left the margin at 1.8rem and lowered the width to 95%. The same goes for the comment below this one.